daemon/lib/swaps/40swap.go

```diff
@@ -0,0 +1,88 @@
+package swaps
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/40acres/40swap/daemon/api"
+	"github.com/40acres/40swap/daemon/database/models"
+)
+
+type FourtySwapClient struct {
```

Comment:
It can be `Client`, there's no more swap clients to be implemented here.

Reply:
Done 🏁